go/obscuronode/host/host.go

```diff
@@ -23,11 +23,6 @@ type AggregatorCfg struct {
 	ClientRPCTimeoutSecs uint64
 }
 
-type L2Network interface {
```

Comment:
can't we keep having this interface (or something similar)?

I think it's useful, as we could have an InProcessImplementation of it, which we could use for a fast development simulation.

Reply:
I think the same can be achieved by passing in an `InProcessP2P` to all the nodes, which would then skip the P2P part.

Reply:
ok. But we need somthing which can either be the real deal or an in-process thing

Reply:
Yep.